cfg/vars.cfg

```diff
@@ -59,10 +59,23 @@ export gitscripts_temp_path="${gitscripts_path}temp/"
 
 
 # formatting
-export H1HL="####################################################################################"
-export H2HL="------------------------------------------------------------------------------------"
-export H3HL="____________________________________________________________________________________"
-export EHL="!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+# export H1HL="####################################################################################"
```

Comment:
I left these in here commented out in case the new `rule` causes unknown bugs it can be easily rolled back by a non-expert.